src/app/components/AddressInputComponent.js

```diff
@@ -94,12 +94,12 @@ const AddressInputComponent = ({
     <div className="row addressInput">
       <div className="row view">
         <div className="col-md-3 label">
-          {label}
+          {labelDisplay || label}
```

Comment:
What is the difference between these?

Reply:
`SubdomainViewContainer` is using `label` to set the new owner.

```
  handleSubmit: newAddress => dispatchProps.handleSubmit(
    stateProps.domain, ownProps.label, newAddress, ownProps.value,
  ),
```

We should display the whole domain

Reply:
ahhh, gotcha. that makes sense.